src/main/java/com/github/takayoshi24/cinema/seans/SeansRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.github.takayoshi24.cinema.seans;
+
+import com.github.takayoshi24.cinema.movie.Movie;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.time.ZonedDateTime;
+import java.util.Optional;
+
+public interface SeansRepository extends JpaRepository<Seans,Long> {
+
+    Optional<Seans> findByMovieAndValidAtAndRoom(Movie movie, ZonedDateTime validAt, Integer room);
```

Comment:
Improve this as discussed.
Also, does this query make sense? Exact ZoneDateTime queries usually are not particularly useful.